
<launch>
  <!-- 启动 test_detector2.py -->
  <node pkg="twist_estimator" type="twist_estimator" name="twist_estimator" output="screen"/>
    <!-- <param name="param_name" value="param_value" /> -->
  <!-- </node> -->

  <param name="config_file" value="$(find twist_estimator)/config/dji2.yaml"/>

  <!-- 启动 combi_radar.py -->
  <node pkg="twist_estimator" type="combi_radar.py" name="combi_radar" output="screen">
    <!-- <param name="param_name" value="param_value" /> -->
  </node>

</launch>
